selfdrive/ui/tests/playsound.cc

```diff
@@ -1,30 +0,0 @@
-#include <QApplication>
```

Comment:
you want to delete this already, I just wrote it :P

Reply:
reverted :D